indico/MaKaC/common/cache.py

```diff
@@ -481,6 +526,10 @@ def __repr__(self):
         return 'GenericCache(%r)' % self._namespace
 
     def _connect(self):
+        """Connect to the CacheClient.
```

Comment:
Just noticed that it's a `_method` - it's usually not really useful writing docstrings for them, they are not part of the public API